index.html

```diff
@@ -0,0 +1,167 @@
+				<a href="#">Logout</a>
+			</div>
+			<div class="navbar nav-btm">
+				<a class="logo" href="#"><img class="logo-img" src="images/logo.png" alt="dsteam logo"></a>
+				<div class="nav-items">
+					<a class="item" href="/about.html">About</a>
+					<a class="item" href="#">Program</a>
+					<a class="item" href="#">Join</a>
+					<a class="item" href="#">Sponsor</a>
+					<a class="item" href="#">News</a>
+					<a class="item" href="#">Campaign</a>
+				</div>
+			</div>
+		</nav>
+		<nav class="navbar fixed-top" id="nav-mobile">
+			<div class="container-fluid">
+				<span class="navbar-toggler border-0" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar"
+					aria-controls="offcanvasNavbar"> <span class="navbar-toggler-icon"></span>
+				</span>
+				<div class="offcanvas offcanvas-start" tabindex="-1" id="offcanvasNavbar"
+					aria-labelledby="offcanvasNavbarLabel">
+					<div class="offcanvas-header">
+						<button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"></button>
+					</div>
+					<div class="offcanvas-body">
+						<ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+							<li class="nav-item"> <a class="nav-link lato" href="/index.html">Home</a> </li>
+							<li class="nav-item"> <a class="nav-link lato" href="/about.html">About</a> </li>
+						</ul>
+					</div>
+				</div>
+			</div>
+		</nav>
+	</header>
+
+	<section id="home" class="m-0">
+		<div class="">
+			<h2 class="color-red">"Hello! Hackers"</h2>
+			<h1 class="cocogoose title" id="">DSTEAM<br>Specialists in computer security <br>2023</h1>
+		</div>
+		<h6 class="greet-p ">We are dedicated to providing services and solutions related to information security and
+			Information Technology. Learn with us, learn about our Courses!</h6>
+		<div class="pt-3 pb-3">
+			<h3 class="cocogoose">2016 ~ 2023 PROVIDING INFO SEC</h3>
+			<h4>@ Medellin - Colombia</h4>
+		</div>
+	</section>
+
+	<section id="services" class="p-5">
+
+		<h2 class="color-white">Our Services</h2>
+		<div class="div-red-under-text"></div>
+
+		<div class="row pt-5 g-3" id="services-wrapper">
+			<div class="card align-items-center p-2 service-card">
+				<div class="p-1"><img src="./icons/pngwing.com.png" class="card-img invert p-1" alt="..."></div>
+				<h5 class="card-title p-1 color-red">Pen Test</h5>
+				<p class="card-text p-1 color-white">Evaluates the security of a computer system or network by
+					simulating an attack from a malicious source.</p>
+			</div>
+			<div class="card align-items-center p-2 service-card">
+				<div class="p-1"> <img src="./icons/pngwing.com (1).png" class="card-img invert p-1" alt="..."></div>
+				<h5 class="card-title p-1 color-red">S-Learning</h5>
+				<p class="card-text p-1 color-white">Online education and training in cybersecurity, learn about the latest
+					tools for protecting digital assets. </p>
+			</div>
+			<div class="card align-items-center p-2 service-card">
```

Comment:
- [ ] I would kindly suggest you give your images an alternative text for easy convention and accessibility. Currently some of your images misses an alternative text.

Reply:
Thanks,  It's done.